### ISSUE TITLE
InternalServerError: unhandled error while calling interpret_backend_error()

### ISSUE THREAD

**Issue by jaclarke** [edgedb/edgedb#5555]
- EdgeDB Version: 4.0-dev.7515+local

Steps to Reproduce:

```
testdb> create type Test {};
OK: CREATE TYPE
testdb> create type Test2 {};
OK: CREATE TYPE
testdb> insert Test2;
{default::Test2 {id: 97e6be22-fbe2-11ed-a45d-1b7647f6e999}}
testdb> alter type Test2 {
.......   create required multi link test: Test;
....... };
Error: InternalServerError: unhandled error while calling interpret_backend_error(); run with EDGEDB_DEBUG_SERVER to debug.
  Hint: This is most likely a bug in EdgeDB. Please consider opening an issue ticket at https://github.com/edgedb/edgedb/issues/new?template=bug_report.md
```

<details>
<summary>Server Logs:</summary>
<pre>
ERROR 19474 2023-05-26T17:30:44.553 postgres: missing value for required property
INFO 19474 2023-05-26T17:30:44.553 postgres: {"object_id": "97e6be22-fbe2-11ed-a45d-1b7647f6e999"}
INFO 19474 2023-05-26T17:30:44.553 postgres: 
INFO 19474 2023-05-26T17:30:44.553 postgres: PL/pgSQL function raise(anyelement,text,text,text,text,text,text,text,text,text) line 3 at RAISE
INFO 19474 2023-05-26T17:30:44.553 postgres:    SQL statement "SELECT
INFO 19474 2023-05-26T17:30:44.554 postgres:        edgedb.raise(
INFO 19474 2023-05-26T17:30:44.554 postgres:            NULL::text,
INFO 19474 2023-05-26T17:30:44.554 postgres:            'not_null_violation',
INFO 19474 2023-05-26T17:30:44.554 postgres:            msg => 'missing value for required property',
INFO 19474 2023-05-26T17:30:44.554 postgres:            detail => '{"object_id": "' || id || '"}',
INFO 19474 2023-05-26T17:30:44.555 postgres:            "column" => 'a98f98d7-fbe2-11ed-a3b8-d78d2df3440c'
INFO 19474 2023-05-26T17:30:44.555 postgres:        )
INFO 19474 2023-05-26T17:30:44.555 postgres:    FROM edgedbpub."82791a26-fbe2-11ed-8e6f-ad1b8429903e"
INFO 19474 2023-05-26T17:30:44.555 postgres:    WHERE id != ALL (SELECT source FROM edgedbpub."a98f98d7-fbe2-11ed-a3b8-d78d2df3440c")
INFO 19474 2023-05-26T17:30:44.555 postgres:    LIMIT 1"
INFO 19474 2023-05-26T17:30:44.555 postgres:    PL/pgSQL function inline_code_block line 84 at SQL statement
INFO 19474 2023-05-26T17:30:44.555 postgres: DO LANGUAGE plpgsql $__$
INFO 19474 2023-05-26T17:30:44.555 postgres:    DECLARE
INFO 19474 2023-05-26T17:30:44.555 postgres:        _dummy_text text;
INFO 19474 2023-05-26T17:30:44.555 postgres:    BEGIN
INFO 19474 2023-05-26T17:30:44.555 postgres: 
INFO 19474 2023-05-26T17:30:44.555 postgres:                    SELECT
INFO 19474 2023-05-26T17:30:44.556 postgres:                        edgedb.raise_on_not_null(
INFO 19474 2023-05-26T17:30:44.556 postgres:                            (SELECT NULLIF(
INFO 19474 2023-05-26T17:30:44.556 postgres:                                (SELECT
INFO 19474 2023-05-26T17:30:44.556 postgres:                                    version::text
INFO 19474 2023-05-26T17:30:44.556 postgres:                                FROM
INFO 19474 2023-05-26T17:30:44.556 postgres:                                    edgedb."_SchemaSchemaVersion"
INFO 19474 2023-05-26T17:30:44.556 postgres:                                FOR UPDATE),
INFO 19474 2023-05-26T17:30:44.556 postgres:                                '827b30bf-fbe2-11ed-b5e2-9b835a35c4e5'
INFO 19474 2023-05-26T17:30:44.556 postgres:                            )),
INFO 19474 2023-05-26T17:30:44.556 postgres:                            'serialization_failure',
INFO 19474 2023-05-26T17:30:44.556 postgres:                            msg => (
INFO 19474 2023-05-26T17:30:44.556 postgres:                                'Cannot serialize DDL: '
INFO 19474 2023-05-26T17:30:44.556 postgres:                                || (SELECT version::text FROM
INFO 19474 2023-05-26T17:30:44.557 postgres:                                    edgedb."_SchemaSchemaVersion")
INFO 19474 2023-05-26T17:30:44.557 postgres:                            )
INFO 19474 2023-05-26T17:30:44.557 postgres:                        )
INFO 19474 2023-05-26T17:30:44.557 postgres:                    INTO _dummy_text;
INFO 19474 2023-05-26T17:30:44.557 postgres: 
INFO 19474 2023-05-26T17:30:44.557 postgres:    IF (NOT EXISTS (SELECT
INFO 19474 2023-05-26T17:30:44.557 postgres:            tablename
INFO 19474 2023-05-26T17:30:44.557 postgres:        FROM
INFO 19474 2023-05-26T17:30:44.557 postgres:            pg_catalog.pg_tables
INFO 19474 2023-05-26T17:30:44.557 postgres:        WHERE
INFO 19474 2023-05-26T17:30:44.557 postgres:            schemaname = 'edgedbpub'
INFO 19474 2023-05-26T17:30:44.557 postgres:            AND tablename = 'a98f98d7-fbe2-11ed-a3b8-d78d2df3440c'
INFO 19474 2023-05-26T17:30:44.557 postgres:        ))
INFO 19474 2023-05-26T17:30:44.558 postgres:    THEN
INFO 19474 2023-05-26T17:30:44.558 postgres:        CREATE TABLE edgedbpub."a98f98d7-fbe2-11ed-a3b8-d78d2df3440c" (source uuid NOT NULL , target uuid NOT NULL , UNIQUE (source, target));
INFO 19474 2023-05-26T17:30:44.558 postgres: 
INFO 19474 2023-05-26T17:30:44.558 postgres: 
INFO 19474 2023-05-26T17:30:44.558 postgres:                    CREATE  INDEX "a98f98d7-fbe2-11ed-a3b8-d78d2df3440c_target_key"
INFO 19474 2023-05-26T17:30:44.558 postgres:                        ON edgedbpub."a98f98d7-fbe2-11ed-a3b8-d78d2df3440c"  ((target) NULLS FIRST);
INFO 19474 2023-05-26T17:30:44.558 postgres: 
INFO 19474 2023-05-26T17:30:44.558 postgres:        EXECUTE 'COMMENT ON INDEX edgedbpub."a98f98d7-fbe2-11ed-a3b8-d78d2df3440c_target_key" IS ' ||
INFO 19474 2023-05-26T17:30:44.558 postgres:            quote_literal(E'EdgeDB metadata follows, do not modify.\n' || '{"code": " ((__col__) NULLS FIRST)", "fullname": "a98f98d7-fbe2-11ed-a3b8-d78d2df3440c_target_key"}');
INFO 19474 2023-05-26T17:30:44.558 postgres: 
INFO 19474 2023-05-26T17:30:44.558 postgres:        COMMENT ON TABLE edgedbpub."a98f98d7-fbe2-11ed-a3b8-d78d2df3440c" IS 'link ''test'' of object type ''default::Test2''';
INFO 19474 2023-05-26T17:30:44.558 postgres:    END IF;
INFO 19474 2023-05-26T17:30:44.559 postgres: 
INFO 19474 2023-05-26T17:30:44.559 postgres:    CREATE OR REPLACE VIEW edgedbpub."a98f98d7-fbe2-11ed-a3b8-d78d2df3440c_t" AS
INFO 19474 2023-05-26T17:30:44.559 postgres:                    (SELECT
INFO 19474 2023-05-26T17:30:44.559 postgres:                       "a98f98d7-fbe2-11ed-a3b8-d78d2df3440c".tableoid AS tableoid,
INFO 19474 2023-05-26T17:30:44.559 postgres:        "a98f98d7-fbe2-11ed-a3b8-d78d2df3440c".xmin AS xmin,
INFO 19474 2023-05-26T17:30:44.559 postgres:        "a98f98d7-fbe2-11ed-a3b8-d78d2df3440c".cmin AS cmin,
INFO 19474 2023-05-26T17:30:44.559 postgres:        "a98f98d7-fbe2-11ed-a3b8-d78d2df3440c".xmax AS xmax,
INFO 19474 2023-05-26T17:30:44.559 postgres:        "a98f98d7-fbe2-11ed-a3b8-d78d2df3440c".cmax AS cmax,
INFO 19474 2023-05-26T17:30:44.559 postgres:        "a98f98d7-fbe2-11ed-a3b8-d78d2df3440c".ctid AS ctid
INFO 19474 2023-05-26T17:30:44.559 postgres:                     FROM
INFO 19474 2023-05-26T17:30:44.559 postgres:                       edgedbpub."a98f98d7-fbe2-11ed-a3b8-d78d2df3440c" AS "a98f98d7-fbe2-11ed-a3b8-d78d2df3440c"
INFO 19474 2023-05-26T17:30:44.559 postgres:                    );
INFO 19474 2023-05-26T17:30:44.559 postgres: 
INFO 19474 2023-05-26T17:30:44.559 postgres:    COMMENT ON VIEW edgedbpub."a98f98d7-fbe2-11ed-a3b8-d78d2df3440c_t" IS 'link ''test'' of object type ''default::Test2'' and descendants';
INFO 19474 2023-05-26T17:30:44.560 postgres: 
INFO 19474 2023-05-26T17:30:44.560 postgres:                        WITH
INFO 19474 2023-05-26T17:30:44.560 postgres:                        "source_a992ec09-fbe2-11ed-a0b9-83cf3bc03019" AS (SELECT * FROM edgedbpub."82791a26-fbe2-11ed-8e6f-ad1b8429903e"
INFO 19474 2023-05-26T17:30:44.560 postgres:    WHERE id NOT IN (SELECT source FROM edgedbpub."a98f98d7-fbe2-11ed-a3b8-d78d2df3440c")
INFO 19474 2023-05-26T17:30:44.560 postgres:    ),
INFO 19474 2023-05-26T17:30:44.560 postgres:                        _conv_rel(val,id) AS 
INFO 19474 2023-05-26T17:30:44.560 postgres:        (
INFO 19474 2023-05-26T17:30:44.560 postgres:        (SELECT/*<pg.SelectStmt at 0x7f9ab23bd3c0>*/
INFO 19474 2023-05-26T17:30:44.560 postgres:                "v~1"."id~1" AS "e~1_serialized~1",
INFO 19474 2023-05-26T17:30:44.560 postgres:                "Test2~3"."Test2_value~1" AS "Test2_identity~1"
INFO 19474 2023-05-26T17:30:44.560 postgres:            FROM 
INFO 19474 2023-05-26T17:30:44.560 postgres:                "source_a992ec09-fbe2-11ed-a0b9-83cf3bc03019" AS "source_a992ec09-fbe2-11ed-a0b9-83cf3bc03019~1"
INFO 19474 2023-05-26T17:30:44.560 postgres:                CROSS JOIN LATERAL 
INFO 19474 2023-05-26T17:30:44.560 postgres:                (SELECT/*<pg.SelectStmt at 0x7f9ab1f2b5e0>*/
INFO 19474 2023-05-26T17:30:44.561 postgres:                        "source_a992ec09-fbe2-11ed-a0b9-83cf3bc03019~1".id AS "Test2_value~1"
INFO 19474 2023-05-26T17:30:44.561 postgres:                ) AS "Test2~3"
INFO 19474 2023-05-26T17:30:44.561 postgres:                CROSS JOIN (SELECT (NULL)::uuid AS "id~1"
INFO 19474 2023-05-26T17:30:44.561 postgres:                    WHERE
INFO 19474 2023-05-26T17:30:44.561 postgres:                        FALSE) AS "v~1"
INFO 19474 2023-05-26T17:30:44.561 postgres:            WHERE
INFO 19474 2023-05-26T17:30:44.561 postgres:                ("v~1"."id~1" IS NOT NULL)
INFO 19474 2023-05-26T17:30:44.561 postgres:        ))
INFO 19474 2023-05-26T17:30:44.561 postgres:                        INSERT INTO edgedbpub."a98f98d7-fbe2-11ed-a3b8-d78d2df3440c" (source, target)
INFO 19474 2023-05-26T17:30:44.561 postgres:                        (SELECT id, val FROM _conv_rel WHERE val IS NOT NULL);
INFO 19474 2023-05-26T17:30:44.562 postgres: 
INFO 19474 2023-05-26T17:30:44.562 postgres:    SELECT
INFO 19474 2023-05-26T17:30:44.562 postgres:        edgedb.raise(
INFO 19474 2023-05-26T17:30:44.562 postgres:            NULL::text,
INFO 19474 2023-05-26T17:30:44.562 postgres:            'not_null_violation',
INFO 19474 2023-05-26T17:30:44.562 postgres:            msg => 'missing value for required property',
INFO 19474 2023-05-26T17:30:44.563 postgres:            detail => '{"object_id": "' || id || '"}',
INFO 19474 2023-05-26T17:30:44.563 postgres:            "column" => 'a98f98d7-fbe2-11ed-a3b8-d78d2df3440c'
INFO 19474 2023-05-26T17:30:44.563 postgres:        )
INFO 19474 2023-05-26T17:30:44.564 postgres:    FROM edgedbpub."82791a26-fbe2-11ed-8e6f-ad1b8429903e"
INFO 19474 2023-05-26T17:30:44.564 postgres:    WHERE id != ALL (SELECT source FROM edgedbpub."a98f98d7-fbe2-11ed-a3b8-d78d2df3440c")
INFO 19474 2023-05-26T17:30:44.564 postgres:    LIMIT 1
INFO 19474 2023-05-26T17:30:44.564 postgres:    INTO _dummy_text;
INFO 19474 2023-05-26T17:30:44.564 postgres: 
INFO 19474 2023-05-26T17:30:44.564 postgres:    IF (EXISTS (SELECT
INFO 19474 2023-05-26T17:30:44.564 postgres:            viewname
INFO 19474 2023-05-26T17:30:44.564 postgres:        FROM
INFO 19474 2023-05-26T17:30:44.564 postgres:            pg_catalog.pg_views
INFO 19474 2023-05-26T17:30:44.565 postgres:        WHERE
INFO 19474 2023-05-26T17:30:44.565 postgres:            schemaname = 'edgedbpub'
INFO 19474 2023-05-26T17:30:44.565 postgres:            AND viewname = 'a98f98d7-fbe2-11ed-a3b8-d78d2df3440c_t'
INFO 19474 2023-05-26T17:30:44.565 postgres:        ))
INFO 19474 2023-05-26T17:30:44.565 postgres:    THEN
INFO 19474 2023-05-26T17:30:44.565 postgres:        DROP VIEW edgedbpub."a98f98d7-fbe2-11ed-a3b8-d78d2df3440c_t";
INFO 19474 2023-05-26T17:30:44.565 postgres:    END IF;
INFO 19474 2023-05-26T17:30:44.566 postgres: 
INFO 19474 2023-05-26T17:30:44.566 postgres:    CREATE  VIEW edgedbpub."a98f98d7-fbe2-11ed-a3b8-d78d2df3440c_t" AS
INFO 19474 2023-05-26T17:30:44.566 postgres:                    (SELECT
INFO 19474 2023-05-26T17:30:44.566 postgres:                       "a98f98d7-fbe2-11ed-a3b8-d78d2df3440c".tableoid AS tableoid,
INFO 19474 2023-05-26T17:30:44.566 postgres:        "a98f98d7-fbe2-11ed-a3b8-d78d2df3440c".xmin AS xmin,
INFO 19474 2023-05-26T17:30:44.566 postgres:        "a98f98d7-fbe2-11ed-a3b8-d78d2df3440c".cmin AS cmin,
INFO 19474 2023-05-26T17:30:44.566 postgres:        "a98f98d7-fbe2-11ed-a3b8-d78d2df3440c".xmax AS xmax,
INFO 19474 2023-05-26T17:30:44.566 postgres:        "a98f98d7-fbe2-11ed-a3b8-d78d2df3440c".cmax AS cmax,
INFO 19474 2023-05-26T17:30:44.566 postgres:        "a98f98d7-fbe2-11ed-a3b8-d78d2df3440c".ctid AS ctid,
INFO 19474 2023-05-26T17:30:44.566 postgres:        "a98f98d7-fbe2-11ed-a3b8-d78d2df3440c".target AS target,
INFO 19474 2023-05-26T17:30:44.566 postgres:        "a98f98d7-fbe2-11ed-a3b8-d78d2df3440c".source AS source
INFO 19474 2023-05-26T17:30:44.566 postgres:                     FROM
INFO 19474 2023-05-26T17:30:44.567 postgres:                       edgedbpub."a98f98d7-fbe2-11ed-a3b8-d78d2df3440c" AS "a98f98d7-fbe2-11ed-a3b8-d78d2df3440c"
INFO 19474 2023-05-26T17:30:44.567 postgres:                    );
INFO 19474 2023-05-26T17:30:44.567 postgres: 
INFO 19474 2023-05-26T17:30:44.567 postgres:    COMMENT ON VIEW edgedbpub."a98f98d7-fbe2-11ed-a3b8-d78d2df3440c_t" IS 'link ''test'' of object type ''default::Test2'' and descendants';
INFO 19474 2023-05-26T17:30:44.567 postgres: 
INFO 19474 2023-05-26T17:30:44.567 postgres:    CREATE OR REPLACE FUNCTION edgedbpub."7d7b54ef-fbe2-11ed-b9b4-69abb3348028_target-del-imm-otl-f"()
INFO 19474 2023-05-26T17:30:44.567 postgres:    RETURNS  trigger
INFO 19474 2023-05-26T17:30:44.567 postgres:    AS $____funcbody____$
INFO 19474 2023-05-26T17:30:44.567 postgres:    DECLARE
INFO 19474 2023-05-26T17:30:44.567 postgres:        link_type_id uuid;
INFO 19474 2023-05-26T17:30:44.567 postgres:        srcid uuid;
INFO 19474 2023-05-26T17:30:44.567 postgres:        tgtid uuid;
INFO 19474 2023-05-26T17:30:44.567 postgres:        linkname text;
INFO 19474 2023-05-26T17:30:44.567 postgres:        endname text;
INFO 19474 2023-05-26T17:30:44.567 postgres:        _dummy_text text;
INFO 19474 2023-05-26T17:30:44.567 postgres:    BEGIN
INFO 19474 2023-05-26T17:30:44.567 postgres:        SELECT
INFO 19474 2023-05-26T17:30:44.567 postgres:        q.__sobj_id__, q.source, q.target
INFO 19474 2023-05-26T17:30:44.568 postgres:        INTO link_type_id, srcid, tgtid
INFO 19474 2023-05-26T17:30:44.568 postgres:    FROM
INFO 19474 2023-05-26T17:30:44.568 postgres:        ((SELECT
INFO 19474 2023-05-26T17:30:44.568 postgres:        'a98f98d7-fbe2-11ed-a3b8-d78d2df3440c'::uuid AS __sobj_id__,
INFO 19474 2023-05-26T17:30:44.568 postgres:        source as source,
INFO 19474 2023-05-26T17:30:44.568 postgres:        target as target
INFO 19474 2023-05-26T17:30:44.568 postgres:    FROM edgedbpub."a98f98d7-fbe2-11ed-a3b8-d78d2df3440c_t")
INFO 19474 2023-05-26T17:30:44.568 postgres:    ) as q
INFO 19474 2023-05-26T17:30:44.568 postgres:    WHERE
INFO 19474 2023-05-26T17:30:44.568 postgres:        q.target = OLD.id
INFO 19474 2023-05-26T17:30:44.568 postgres:    LIMIT 1;
INFO 19474 2023-05-26T17:30:44.568 postgres: 
INFO 19474 2023-05-26T17:30:44.568 postgres:    IF FOUND THEN
INFO 19474 2023-05-26T17:30:44.568 postgres:        SELECT
INFO 19474 2023-05-26T17:30:44.569 postgres:            edgedb.shortname_from_fullname(link.name),
INFO 19474 2023-05-26T17:30:44.569 postgres:            edgedb._get_schema_object_name(link.source)
INFO 19474 2023-05-26T17:30:44.569 postgres:            INTO linkname, endname
INFO 19474 2023-05-26T17:30:44.569 postgres:        FROM
INFO 19474 2023-05-26T17:30:44.569 postgres:            edgedb."_SchemaLink" AS link
INFO 19474 2023-05-26T17:30:44.569 postgres:        WHERE
INFO 19474 2023-05-26T17:30:44.569 postgres:            link.id = link_type_id;
INFO 19474 2023-05-26T17:30:44.569 postgres:        RAISE foreign_key_violation
INFO 19474 2023-05-26T17:30:44.569 postgres:            USING
INFO 19474 2023-05-26T17:30:44.569 postgres:                TABLE = TG_TABLE_NAME,
INFO 19474 2023-05-26T17:30:44.569 postgres:                SCHEMA = TG_TABLE_SCHEMA,
INFO 19474 2023-05-26T17:30:44.569 postgres:                MESSAGE = 'deletion of default::Test (' || tgtid
INFO 19474 2023-05-26T17:30:44.569 postgres:                    || ') is prohibited by link target policy',
INFO 19474 2023-05-26T17:30:44.570 postgres:                DETAIL = 'Object is still referenced in link '
INFO 19474 2023-05-26T17:30:44.570 postgres:                    || linkname || ' of ' || endname || ' ('
INFO 19474 2023-05-26T17:30:44.570 postgres:                    || srcid || ').';
INFO 19474 2023-05-26T17:30:44.570 postgres:    END IF;
INFO 19474 2023-05-26T17:30:44.570 postgres: 
INFO 19474 2023-05-26T17:30:44.570 postgres:        RETURN OLD;
INFO 19474 2023-05-26T17:30:44.570 postgres:    END;
INFO 19474 2023-05-26T17:30:44.570 postgres:    $____funcbody____$
INFO 19474 2023-05-26T17:30:44.570 postgres:    LANGUAGE plpgsql VOLATILE  PARALLEL UNSAFE;
INFO 19474 2023-05-26T17:30:44.570 postgres: 
INFO 19474 2023-05-26T17:30:44.570 postgres:    IF (NOT EXISTS (SELECT
INFO 19474 2023-05-26T17:30:44.570 postgres:            tg.tgname
INFO 19474 2023-05-26T17:30:44.570 postgres:        FROM
INFO 19474 2023-05-26T17:30:44.570 postgres:            pg_catalog.pg_trigger tg
INFO 19474 2023-05-26T17:30:44.571 postgres:            INNER JOIN pg_catalog.pg_class tab
INFO 19474 2023-05-26T17:30:44.571 postgres:                ON (tab.oid = tg.tgrelid)
INFO 19474 2023-05-26T17:30:44.571 postgres:            INNER JOIN pg_catalog.pg_namespace ns
INFO 19474 2023-05-26T17:30:44.571 postgres:                ON (ns.oid = tab.relnamespace)
INFO 19474 2023-05-26T17:30:44.571 postgres:        WHERE
INFO 19474 2023-05-26T17:30:44.571 postgres:            tab.relname = '7d7b54ef-fbe2-11ed-b9b4-69abb3348028'
INFO 19474 2023-05-26T17:30:44.571 postgres:            AND ns.nspname = 'edgedbpub'
INFO 19474 2023-05-26T17:30:44.571 postgres:            AND tg.tgname = 't_7d7b54ef-fbe2-11ed-b9b4-69abb3348028_target-del-imm-otl-t'
INFO 19474 2023-05-26T17:30:44.571 postgres:        ))
INFO 19474 2023-05-26T17:30:44.571 postgres:    THEN
INFO 19474 2023-05-26T17:30:44.571 postgres:        CREATE CONSTRAINT TRIGGER "t_7d7b54ef-fbe2-11ed-b9b4-69abb3348028_target-del-imm-otl-t" after delete
INFO 19474 2023-05-26T17:30:44.571 postgres:               ON edgedbpub."7d7b54ef-fbe2-11ed-b9b4-69abb3348028"
INFO 19474 2023-05-26T17:30:44.571 postgres:           
INFO 19474 2023-05-26T17:30:44.572 postgres:               FOR EACH row 
INFO 19474 2023-05-26T17:30:44.572 postgres:               EXECUTE PROCEDURE edgedbpub."7d7b54ef-fbe2-11ed-b9b4-69abb3348028_target-del-imm-otl-f"();
INFO 19474 2023-05-26T17:30:44.572 postgres: 
INFO 19474 2023-05-26T17:30:44.572 postgres:        EXECUTE 'COMMENT ON TRIGGER "t_7d7b54ef-fbe2-11ed-b9b4-69abb3348028_target-del-imm-otl-t" ON edgedbpub."7d7b54ef-fbe2-11ed-b9b4-69abb3348028" IS ' ||
INFO 19474 2023-05-26T17:30:44.572 postgres:            quote_literal(E'EdgeDB metadata follows, do not modify.\n' || '{"ddl:inherit": true}');
INFO 19474 2023-05-26T17:30:44.572 postgres:    END IF;
INFO 19474 2023-05-26T17:30:44.572 postgres: 
INFO 19474 2023-05-26T17:30:44.572 postgres: 
INFO 19474 2023-05-26T17:30:44.572 postgres:                    PERFORM edgedb.__rh_09bef9deed149c843e49fee68e6c914d2bb4099d('"__schema_version__"', '"a9900385-fbe2-11ed-afe7-595f10a5067b"');
INFO 19474 2023-05-26T17:30:44.572 postgres: 
INFO 19474 2023-05-26T17:30:44.572 postgres: 
INFO 19474 2023-05-26T17:30:44.572 postgres:                    PERFORM edgedb.__rh_68504f7ff2653d39c8375ac8e4c610b767b80c53('false', 'null', '"DDLStatement"', '"a98f83de-fbe2-11ed-ac69-07cfbadf9e18"', 'false', 'null', '"m1zutxt35hk4bfnk2p3deo5vhbyvo5in4ky7qio4fwitxzey2qnvaa"', '"m1zutxt35hk4bfnk2p3deo5vhbyvo5in4ky7qio4fwitxzey2qnvaa"', '["827904a6-fbe2-11ed-b52e-5b39cdc0cd52"]', '"SET generated_by := (schema::MigrationGeneratedBy.DDLStatement);\nALTER TYPE Test2 {\n    CREATE REQUIRED MULTI LINK test: Test;\n};"');
INFO 19474 2023-05-26T17:30:44.572 postgres: 
INFO 19474 2023-05-26T17:30:44.572 postgres: 
INFO 19474 2023-05-26T17:30:44.573 postgres:                    PERFORM edgedb.__rh_94cd4e2da646997d5229e9ab2a1fb72aca4f464c('"default::Test2"', 'null', '["329266dc-f89c-11ed-8d78-a168106837c1"]', '["329266dc-f89c-11ed-8d78-a168106837c1"]', 'false', '"Many"', 'null', 'null', 'null', 'null', 'null', 'null', 'null', '"a98f98d7-fbe2-11ed-a3b8-d78d2df3440c"', 'null', 'false', 'null', '"default::__|test@default|Test2"', '"test"', 'null', 'null', 'true', 'null', 'true', '"82791a26-fbe2-11ed-8e6f-ad1b8429903e"', '"7d7b54ef-fbe2-11ed-b9b4-69abb3348028"');
INFO 19474 2023-05-26T17:30:44.573 postgres: 
INFO 19474 2023-05-26T17:30:44.573 postgres: 
INFO 19474 2023-05-26T17:30:44.573 postgres:                    PERFORM edgedb.__rh_25b8c387e33569a32e693b8155344e733ee7dcf1('"default::__|test@default|Test2"', 'null', '["3291c22e-f89c-11ed-8461-47561c716fb6", "328fb7c7-f89c-11ed-a87d-a746bf134caf"]', '["3291c22e-f89c-11ed-8461-47561c716fb6"]', 'false', '"One"', 'null', 'null', 'null', 'null', 'null', 'null', 'null', '"a98fc3bc-fbe2-11ed-9349-1f1867970e81"', 'null', 'false', 'null', '"default::__|target@default|__||test&default||Test2"', '"target"', 'true', 'true', 'false', '"a98f98d7-fbe2-11ed-a3b8-d78d2df3440c"', '"7d7b54ef-fbe2-11ed-b9b4-69abb3348028"');
INFO 19474 2023-05-26T17:30:44.573 postgres: 
INFO 19474 2023-05-26T17:30:44.573 postgres: 
INFO 19474 2023-05-26T17:30:44.573 postgres:                    PERFORM edgedb.__rh_25b8c387e33569a32e693b8155344e733ee7dcf1('"default::__|test@default|Test2"', 'null', '["3291078d-f89c-11ed-9d6f-49203e8ad6b6", "328fb7c7-f89c-11ed-a87d-a746bf134caf"]', '["3291078d-f89c-11ed-9d6f-49203e8ad6b6"]', 'false', '"One"', 'null', 'null', 'null', 'null', 'null', 'null', 'null', '"a98fe8d9-fbe2-11ed-b44f-0bd737b1d39f"', 'null', 'false', 'null', '"default::__|source@default|__||test&default||Test2"', '"source"', 'true', 'true', 'true', '"a98f98d7-fbe2-11ed-a3b8-d78d2df3440c"', '"82791a26-fbe2-11ed-8e6f-ad1b8429903e"');
INFO 19474 2023-05-26T17:30:44.573 postgres:    END;
INFO 19474 2023-05-26T17:30:44.573 postgres:    $__$;
======================================== Exception occurred: missing value for required property =========================================

1. edb.server.pgcon.errors.BackendError: missing value for required property

--------------------------------------------------------------- Traceback ----------------------------------------------------------------

    edb/server/protocol/binary.pyx, line 1026, in edb.server.protocol.binary.EdgeConnection.main_step
        
    edb/server/protocol/binary.pyx, line 981, in execute
        
    edb/server/protocol/binary.pyx, line 786, in _execute
        
    edb/server/protocol/execute.pyx, line 163, in execute
        > ???
    edb/server/protocol/execute.pyx, line 107, in edb.server.protocol.execute.execute
        > ???
    edb/server/pgcon/pgcon.pyx, line 1306, in parse_execute
        
    edb/server/pgcon/pgcon.pyx, line 1259, in _parse_execute
        

edb.server.pgcon.errors.BackendError: missing value for required property
ERROR 19423 2023-05-26T17:30:44.573 asyncio: an error in edgedb protocol
protocol: <edb.server.protocol.binary.EdgeConnectionBackwardsCompatible object at 0x7fc3346fd5a0>
transport: <uvloop.loop._SSLProtocolTransport object at 0x7fc33af92800>
INFO 19423 2023-05-26T17:32:17.472 edb.server: Connection discarded to backend database: edgedb
</pre>
</details>

**Comment by MrFoxPro** [comment]
I encountered same issue on `4.0+d568f08` when applying migration

**Comment by msullivan** [comment]
Fixed by #6852
